Sources/Hummingbird/Middleware/LogRequestMiddleware.swift

```diff
@@ -72,28 +72,28 @@ public struct LogRequestsMiddleware<Context: RequestContext>: RouterMiddleware {
                 level: self.logLevel,
                 "Request",
                 metadata: [
-                    "hb_uri": .stringConvertible(request.uri),
-                    "hb_method": .string(request.method.rawValue),
+                    "hb.uri": .stringConvertible(request.uri),
+                    "hb.method": .string(request.method.rawValue),
```

Comment:
```suggestion
                    "http.request.path": .stringConvertible(request.uri),
                    "http.request.method": .string(request.method.rawValue),
```

Reply:
Using `hb` prefix still but followed the rest of your suggestion